lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
```

Comment:
Nothing bad happens if you pass `clobber` in a `1.3.devxxx` version because it is not removed, just hidden. Unlike if you pass `overwrite` before it was introduced. It is probably a are case and people who run a dev version probably don't mind updating.